src/gsy_e/setup/strategy_tests/future_market_strategy.py

```diff
@@ -0,0 +1,97 @@
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+"""
+import os
+from gsy_framework.constants_limits import ConstSettings
+
+from gsy_e.constants import FutureTemplateStrategiesConstants
+from gsy_e.models.area import Area
+from gsy_e.models.strategy.external_strategies.load import (
+    LoadHoursStrategy,
+)
+from gsy_e.models.strategy.external_strategies.pv import PVStrategy
+from gsy_e.models.strategy.external_strategies.storage import (
+    StorageStrategy,
+)
+
+current_dir = os.path.dirname(__file__)
+
+
+def get_setup(config):
+    ConstSettings.MASettings.MARKET_TYPE = 2
+    ConstSettings.FutureMarketSettings.FUTURE_MARKET_DURATION_HOURS = (
+        get_var_from_env("FUTURE_MARKET_DURATION_HOURS", 1)
+    )
+    ConstSettings.FutureMarketSettings.FUTURE_MARKET_CLEARING_INTERVAL_MINUTES = (
+        get_var_from_env("FUTURE_MARKET_CLEARING_INTERVAL_MINUTES", 5)
+    )
+    FutureTemplateStrategiesConstants.UPDATE_INTERVAL_MIN = get_var_from_env(
+        "UPDATE_INTERVAL_MIN", 5
+    )
+    FutureTemplateStrategiesConstants.INITIAL_BUYING_RATE = get_var_from_env(
+        "INITIAL_BUYING_RATE", 0
+    )
+    FutureTemplateStrategiesConstants.FINAL_BUYING_RATE = get_var_from_env(
+        "FINAL_BUYING_RATE", 50
+    )
+    FutureTemplateStrategiesConstants.INITIAL_SELLING_RATE = get_var_from_env(
+        "INITIAL_SELLING_RATE", 50
+    )
+    FutureTemplateStrategiesConstants.FINAL_SELLING_RATE = get_var_from_env(
+        "FINAL_SELLING_RATE", 0
+    )
+
+    area = Area(
+        "Grid-Community",
+        [
+            Area("House 1",
+                 [
+                     Area(
+                         "Load",
+                         strategy=LoadHoursStrategy(
+                             avg_power_W=100000,
+                             hrs_per_day=24,
+                             hrs_of_day=list(range(0, 24)),
+                             initial_buying_rate=3,
+                             final_buying_rate=24,
+                         ),
+                     ),
+                     Area(
+                         "PV", strategy=PVStrategy(
+                             panel_count=1, initial_selling_rate=47, final_selling_rate=25)
+                     ),
+                     Area(
+                         "ESS",
+                         strategy=StorageStrategy(
+                             initial_soc=50, battery_capacity_kWh=1000000,
+                             initial_selling_rate=47, final_selling_rate=25,
+                             initial_buying_rate=3, final_buying_rate=24)
+                     )
+                 ]
+                 )
+        ], config=config,
+    )
+
+    return area
+
+
+def get_var_from_env(var_name: str, default_val: float):
+    """Read variable from env and parse as string"""
+    var = os.environ.get(var_name)
+    if var is not None:
+        return float(var)
+    return default_val
```

Comment:
I assume that you did not use this instead:
```
def get_var_from_env(var_name: str, default_val: float):
    """Read variable from env and parse as string"""
    var = float(os.environ.get(var_name, default_val) or default_val)
```
in order to cover the case that the variable has a value of 0 (in which case you do not want to fall back to the default value)?

Reply:
I did not see this part until now. The reasoning you describe make sense.